vehicle/homeassistant.go

```diff
@@ -224,13 +224,16 @@ func (v *HomeAssistant) status(sensor string) (api.ChargeStatus, error) {
 		"ready":               api.StatusB,
 		"plugged":             api.StatusB,
 		"charging_completed":  api.StatusB,
+		"initialising":        api.StatusB,
 		"a":                   api.StatusA,
 		"disconnected":        api.StatusA,
 		"off":                 api.StatusA,
 		"none":                api.StatusA,
 		"unavailable":         api.StatusA,
 		"unknown":             api.StatusA,
 		"notreadyforcharging": api.StatusA,
+		"not_plugged":         api.StatusA,
+		"charging_error":      api.StatusA,
```

Comment:
I'm not 100% certain this is correct. Maybe B state would be correct for charging_error? As the charger is reporting that it's connected, maybe the car should be reporting the same? What do you think?

Reply:
Error state are always that- neither A-C, so just leave out.